### ISSUE TITLE
Front-end highlighter redesign: issue pagination screen reader announcements should be improved

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1629]
When using the next/previous buttons in the front-end highlighter to move through issues, the only screen reader announcement is the number being shown, I.e., "4 of 17".

<img src="https://github.com/user-attachments/assets/3f3dd98b-ba42-4321-b0ae-94764aec38c3 " alt="Image" width="1098" data-linear-height="283" />

This doesn't provide enough context if the user is looking for a specific issue. It would be most ideal if the issue title was also read. There's an existing container with aria-live on it already:

```
<span class="edac-highlight-panel-controls-pagination" id="edac-highlight-pagination" aria-live="polite">4 of 18</span>
```

Perhaps this can be solved by adding the issue title as screen reader text inside this span? Or, add the title in the status container requested in [PRO-733](https://linear.app/equalize-digital/issue/PRO-733/front-end-highlighter-redesign-missing-screen-reader-announcements).
